omnibus/config/software/datadog-agent-finalize.rb

```diff
@@ -59,7 +59,7 @@
                 link "#{install_dir}/embedded/bin/python3", "#{install_dir}/embedded/bin/python"
 
                 delete "#{install_dir}/embedded/bin/2to3"
-                link "#{install_dir}/embedded/bin/2to3-3.9", "#{install_dir}/embedded/bin/2to3"
```

Comment:
out of curiosity, do we need to ship 2to3? This feels more like a dev tool than something we'd need at runtime

Reply:
Very good question. I'll ask my team and will open another PR to drop that part if possible 🙂

Reply:
Thanks! This is by no mean blocking for this PR